moirae/estimators/online/joint.py

```diff
@@ -116,16 +137,24 @@ def initialize_unscented_kalman_filter(cls,
             joint_hidden_covariance = block_diag(covariance_transient, covariance_asoh)
         else:
             joint_hidden_covariance = covariance_joint
+        # Convert covariance!
+        joint_hidden_covariance = asoh_normalizer.inverse_transform_covariance(
+            transformed_covariance=joint_hidden_covariance)
         joint_initial_hidden = MultivariateGaussian(mean=joint_hidden_mean[0, :], covariance=joint_hidden_covariance)
         # Initial controls
-        initial_controls = convert_vals_model_to_filter(model_quantities=initial_inputs,
-                                                        uncertainty_matrix=inputs_uncertainty)
+        initial_controls = convert_vals_model_to_filter(
+            model_quantities=joint_model._convert_to_control_samples(model_control_samples=initial_inputs),
```

Comment:
As above, do not call private methods and let's also not have functions that are just 1:1 mappings to calling a function from another class.

Reply:
Of course, idk what I was thinking. Fixing these now.